src/styles/foundation/_layout.scss

```diff
@@ -53,7 +83,7 @@ $layout-width-data: (
   @if type-of($fetched-value) {
     @return $fetched-value;
   } @else {
-    @error 'Column `#{$name} - #{$value}` not found. Available columns: #{available-names($layout-width-data)}';
+    @error 'Column `#{$name} - #{$value}` not found. Available columns: #{available-layout-names()}';
```

Comment:
I'd be pretty tempted to hardcode the string of what the function returns. We'd need to update it if we modify the `$layout-width-data` but I suspect when that happens the whole thing shall  be radically reworked anyway.

Example error:

```
Error: Column `primary - max` not found. Available columns:
       - primary (min, max)
       - secondary (min, max)
       - one-half-width (base)
       - one-third-width (base)
       - nav (base)
       - page-with-nav (base)
       - page-content (not-condensed, partially-condensed)
       - inner-spacing (base)
       - outer-spacing (min, max)
```


Code:

```
@error 'Column `#{$name} - #{$value}` not found. Available columns:\a       - primary (min, max)\a       - secondary (min, max)\a       - one-half-width (base)\a       - one-third-width (base)\a       - nav (base)\a       - page-with-nav (base)\a       - page-content (not-condensed, partially-condensed)\a       - inner-spacing (base)\a       - outer-spacing (min, max)';'
```

Note I'm using `\a` for newlines so that it works in both node-sass and dart-sass per https://github.com/Shopify/polaris-react/pull/3063

Reply:
This looks good to me. Keeps the error message details and removes the function. Thanks Ben!